# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/src/args.yaml
# Commit: 431c5142ff3a
# Full Hash: 431c5142ff3ad102328d90cb29470b20ccd1271e
# Author: Glenn Watson <github@intuitionlibrary.com>
# Date: 2019-06-13 21:50:57
# Regressor Bug: 1558106
# File Overlap Count: 1
# Description:
#   Bug 1558106 - Support picture caching for multiple scroll roots. r=kvark
#   
#   This patch implements the majority of the planned picture caching
#   improvements. It supports most of the functionality required to
#   (as a follow up) support OS compositor integration. It also improves
# ==============================================================================

diff -r 855d557b82a2 -r 431c5142ff3a gfx/wr/wrench/src/args.yaml
--- a/gfx/wr/wrench/src/args.yaml	Wed Jun 12 16:18:22 2019 +0000
+++ b/gfx/wr/wrench/src/args.yaml	Thu Jun 13 04:43:56 2019 +0000
@@ -33,6 +33,9 @@
       short: a
       long: no-subpixel-aa
       help: Disable subpixel aa
+  - no_picture_caching:
+      long: no-picture-caching
+      help: Disable picture caching
   - slow_subpixel:
       long: slow-subpixel
       help: Disable dual source blending